packages/xod-client/src/projectBrowser/containers/ProjectBrowser.jsx

```diff
@@ -105,12 +105,14 @@ class ProjectBrowser extends React.Component {
       <Icon
         key="delete"
         name="trash"
+        title="Delete Patch"
```

Comment:
It’s better to be consistent about tooltip case. In other places, we use the regular case, so this should be “Delete patch”.

The main menu (if you recall it) and button labels is another story. They’re traditionally in “Newspaper Case”.

See Atom for example.